app/javascript/dashboard/routes/dashboard/contacts/components/ImportContacts.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <modal :show.sync="show" :on-close="onClose">
+    <div class="column content-box">
+      <woot-modal-header :header-title="$t('IMPORT_CONTACTS.TITLE')">
+        <p>
+          Import contacts through a CSV file.
```

Comment:
Please use i18n here

Reply:
https://github.com/chatwoot/chatwoot/commit/1dce7151bbefa7938e8c25fb218f3dad176e619e Replaced the harcoded label